<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranscriberWindow</class>
 <widget class="QMainWindow" name="TranscriberWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="stand::gui::MappingView" name="MappingView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="stand::gui::HorizontalPianoView" name="PianoView" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>32</height>
         </size>
        </property>
        <zorder>MappingView</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QTabWidget" name="SettingTabs">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="stand::gui::TranscriberWidget" name="Base">
       <attribute name="title">
        <string>Base</string>
       </attribute>
       <zorder>horizontalFrame_2</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="10" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="AddButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Add Tab</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RemoveButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Tab</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="AnalyzeButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ExitButton">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0">
     <widget class="QProgressBar" name="ProgressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string>%v/%m</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ThreadLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Threads</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="NumThreads">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="horizontalFrame_2">
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="6" column="9">
        <widget class="QPushButton" name="LibraryIconSelector">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QLabel" name="label">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Encode</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QComboBox" name="RootEncode">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Shift-JIS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>EUC-JP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UTF-8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="9">
        <widget class="QPushButton" name="RootSelector">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3" colspan="6">
        <widget class="QLineEdit" name="LibraryWeb">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="6">
        <widget class="QLineEdit" name="LibraryIconPath"/>
       </item>
       <item row="4" column="3" colspan="6">
        <widget class="QLineEdit" name="RootDirectory"/>
       </item>
       <item row="2" column="5">
        <widget class="QLineEdit" name="LibraryAuthor">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Author</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="LibraryName">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="8">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Library Information</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Site</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="RootLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Root</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>478</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stand::gui::HorizontalPianoView</class>
   <extends>QWidget</extends>
   <header>HorizontalPianoView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stand::gui::TranscriberWidget</class>
   <extends>QWidget</extends>
   <header>TranscriberWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stand::gui::MappingView</class>
   <extends>QWidget</extends>
   <header>MappingView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>AddButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWindow</receiver>
   <slot>addTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RemoveButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWindow</receiver>
   <slot>removeTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExitButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AnalyzeButton</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWindow</receiver>
   <slot>pushAnalyze()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RootSelector</sender>
   <signal>clicked()</signal>
   <receiver>TranscriberWindow</receiver>
   <slot>pushRootDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>452</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTab()</slot>
  <slot>removeTab()</slot>
  <slot>pushAnalyze()</slot>
  <slot>pushRootDir()</slot>
  <slot>pushIcon()</slot>
 </slots>
</ui>
